configure.ac

```diff
@@ -5227,12 +5235,69 @@ if test "x$enable_btls" = "xyes"; then
 	AC_SUBST(BTLS_CMAKE_ARGS)
 
 	AC_DEFINE(HAVE_BTLS, 1, [BoringTls is supported])
+	AC_DEFINE(HAVE_TLS, 1, [Unix Tls is supported])
 else
 	enable_btls=no
 fi
 
 AM_CONDITIONAL(BTLS_ANDROID, test x$btls_android = xyes)
 
+dnl ****************
+dnl ***  OpenSSL ***
+dnl ****************
+
+AC_ARG_ENABLE(openssl, [  --disable-openssl             Disable the OpenSSL provider], enable_openssl=$enableval, enable_openssl=$OPENSSL_SUPPORTED)
+
+AM_CONDITIONAL(OPENSSL, test x$enable_openssl = xyes)
+
+if test "x$enable_openssl" = "xyes"; then
+	AC_PATH_PROG(CMAKE, [cmake], [no], [$PATH:/Applications/CMake.app/Contents/bin:/usr/local/bin])
```

Comment:
This conflicts with other existing cmake check (including the one for BTLS) and needs to be moved out of that conditional.